src/notifications/CommandSend.ts

```diff
@@ -17,6 +17,10 @@ class CommandSend extends CommandPolykey {
       binParsers.parseNodeId,
     );
     this.argument('<message>', 'Message to send');
+    this.option(
```

Comment:
I think we have a generic retry command we tried to put in before but it was never merged. Can you check the old PRs?

Reply:
i dont think i can see such PR, was this a closed PR on Polykey CLI?

Reply:
![image](https://github.com/MatrixAI/Polykey-CLI/assets/50583248/49562284-f7b0-4d56-a114-f25e5a217d52)

Reply:
Yes here:

* https://github.com/MatrixAI/Polykey/pull/325#issuecomment-1670912995
* https://github.com/MatrixAI/Polykey-CLI/issues/16

I think the general gist, is that https://github.com/MatrixAI/Polykey-CLI/issues/16 would require refactoring those options sharable options used from `bin/options.ts` and so different commands that needed something like that, it would be re-used.

Reply:
And note how because they are generic, they basically are linear delays, rather than exponential delays, but if we wanted to, it could be:

```
--retry-count     == 0 (default to not retrying)
--retry-interval  == 100 (in ms)
--retry-multipler == 1
```

Although I want to leave the option for infinite retries too. While designing https://github.com/MatrixAI/Polykey/issues/715 - there's an option for user-level cancellation, so that retries go infinite until the user decides there's no point.

So to define sentinel values for them (which should be specced out):

* -1 for infinity, 0 for no retry... etc.

Reply:
Ah alright, this might need changes in PK itself to expose the retry multiplier option

Reply:
Not in PK, just in `src/utils/options.ts` - it stores ALL the generic options to be re-used. Always consult that first before adding new kinds of options.